backend/src/data/migrations/20220902141153_add_status_to_chat_messages_table.ts

```diff
@@ -0,0 +1,26 @@
+import { Knex } from 'knex';
+
+enum TableName {
+  CHAT_MESSAGES = 'chat_messages',
+}
+
+const ColumnName = {
+  STATUS: 'status',
+} as const;
+
+async function up(knex: Knex): Promise<void> {
+  await knex.schema.table(TableName.CHAT_MESSAGES, (table) => {
+    table
+      .enum(ColumnName.STATUS, ['read', 'unread'])
+      .defaultTo('unread')
```

Comment:
```suggestion
      .enum(ColumnName.STATUS, Object.values(ChatMessageStatus))
      .defaultTo(ChatMessageStatus.UNREAD)
```
create this enum in this file